content/docs/tech-insights/add-check/index.md

```diff
@@ -6,19 +6,59 @@ description: Add new check.
 
 After navigating to Tech Insights→ Checks page and clicking ‘Add check’ button, you’ll be asked to describe your Check as below:
```

Comment:
Would be nice to add a space after Tech Insights and before the arrow - obviously not something you've changed but still...